test/tests/components/ws-manager/prebuild_test.go

```diff
@@ -180,8 +180,22 @@ func TestPrebuildWorkspaceTaskFail(t *testing.T) {
 const (
 	prebuildLogPath string = "/workspace/.gitpod"
 	prebuildLog     string = "'🤙 This task ran as a workspace prebuild'"
+	initTask        string = "echo \"some output\" > someFile; sleep 90;"
 )
 
+// TestOpenWorkspaceFromPrebuild
+// - create a prebuild
+// - open the workspace from prebuild
+// - make sure the .git/ folder with correct permission
+// - make sure either one of the condition mets
+//   - the prebuild log message exists
+//   - the init task message exists
+//   - the init task generated file exists
+//
+// - write a new file foobar.txt
+// - stop the workspace
+// - relaunch the workspace
+// - make sure the file foobar.txt exists
```

Comment:
Add the test steps here, so we could more easily understand the test steps without tracing the code.

Reply:
Great!